sdk/src/vm.ts

```diff
@@ -137,10 +143,12 @@ export class VM {
      * @experimental
      */
     getUrl: (): Promise<string | null> => {
-      return this._rdc.request<string | null>({
-        type: 'SDK_GET_PREVIEW_URL',
-        payload: {},
-      });
+      return this._rdc
+        .request<{ url: string } | null>({
+          type: 'SDK_GET_PREVIEW_URL',
+          payload: {},
+        })
+        .then((data) => data?.url ?? null);
```

Comment:
Fix for the issue where we were returning `{ url: string } | null` and not `string | null`.